packages/blockchain-link-utils/src/blockbook.ts

```diff
@@ -1,3 +1,5 @@
+import { Ethereum } from '@everstake/wallet-sdk';
```

Comment:
Would it be possible to import just `selectNetwork` from `Ethereum`?

By this, we take whole lib and it's deps (solana, cosmos,..) into blockbook worker which is just for Bitcoin and EVMs
![Screenshot 2024-07-24 at 15 13 45](https://github.com/user-attachments/assets/f8b45aab-d8cc-44b9-a546-5b2d03b87336)
![Screenshot 2024-07-24 at 15 14 54](https://github.com/user-attachments/assets/99159d27-994a-4ee6-a009-ff1c174af3e5)

Reply:
https://github.com/trezor/trezor-suite/pull/13267/commits/956a4043aa467b08f04408fd8607a17871383d41
I've also added these changes. For now, it can be challenging to tree-shake Web3 effectively because of CommonJS imports in wallet-sdk. However, we can get rid of other chains (solana, etc) for now